app/components/EvolverTree.tsx

```diff
@@ -0,0 +1,72 @@
+import {
```

Comment:
Stub for an experiment in having a file-tree representation of the Evolver config. Not being used anywhere. happy to delete if YAGNI proponents amongst us.